public/style.css

```diff
@@ -88,17 +90,15 @@ body {
 
 .app-container .left .top {
   position: relative;
-  width: 100%;
   height: 80px;
-  padding: 29px;
 }
 
 .app-container .left .top:after {
   position: absolute;
   bottom: 0;
   left: 50%;
   display: block;
-  width: 80%;
+  width: 90%;
   height: 1px;
   content: '';
   background-color: #e6e6e6;
```

Comment:
@dvshsahu can you think of something that we can get rid of this code also, perhaps a `container-fluid` can work(haven't tried it yet).

Reply:
removed the code, now using plain <hr> tag to achieve the same